tests/tests.py

```diff
@@ -0,0 +1,25 @@
+from fastapi.testclient import TestClient
+from app import app
+
+client = TestClient(app)
+
+
```

Comment:
Testy nie będą przechodziły bo na nodzie na którym odpalane są testy nie ma local_auth.py więc nie może się zalogować XD

Reply:
Pamiętam, że do tego doszedłem w końcu wczoraj, ale nie wiedziałem, jak ustawić te envy tutaj - wiesz jak to zrobić, czy szukać?

Reply:
https://docs.github.com/en/actions/reference/encrypted-secrets
To sugeruje tam w zakładach ale nigdy tego nie robiłem